packages/bvaughn-architecture-demo/src/contexts/InspectorContext.ts

```diff
@@ -18,6 +20,7 @@ type InspectHTMLElement = (
 export type InspectorContextType = {
```

Comment:
This change (injecting a param) seems like the right one 👍🏼 

I think maybe we should _rename_ this context at this point though. It's not just 'inspecting' things anymore. It's kind of just a generic adapter between legacy and new architectures.